Makefile

```diff
@@ -34,20 +36,24 @@ help:
 	@echo "usage: make COMMAND"
 	@echo ""
 	@echo "Commands:"
-	@echo "  list-tests               List all the test under the tests directory"
+	@echo "  tests/list               List all the test under the tests directory"
```

Comment:
What do you think about keeping this as `test/list`?

Reply:
Done

Reply:
You forgot to update the help documentation:)